interview/inventory/urls.py

```diff
@@ -1,6 +1,10 @@
 
 from django.urls import path
-from interview.inventory.views import InventoryLanguageListCreateView, InventoryLanguageRetrieveUpdateDestroyView, InventoryListCreateView, InventoryRetrieveUpdateDestroyView, InventoryTagListCreateView, InventoryTagRetrieveUpdateDestroyView, InventoryTypeListCreateView, InventoryTypeRetrieveUpdateDestroyView
+from interview.inventory.views import InventoryLanguageListCreateView, \
```

Comment:
slashes here are considered bad practice. If you are going to separate them onto different lines like this try this format:
from x import (
y,
z,
w
)

Reply:
Changes made.